react_ws_src/src/views/ttt/GameMain.js

```diff
@@ -192,29 +191,15 @@ export default class SetName extends Component {
 //	------------------------	------------------------	------------------------
 
 	turn_comp () {
```

Comment:
I wanted to make minimal changes to this file, which is why I've encapsulated all of the logic into `const cell_id = get_computer_move(cell_vals);`

Everything else, except the animation and state mutation, could be removed in this function.